recipes/cassandra-cpp-driver/all/conanfile.py

```diff
@@ -0,0 +1,137 @@
+import os.path
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class CassandraCppDriverConan(ConanFile):
+    name = "cassandra-cpp-driver"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://docs.datastax.com/en/developer/cpp-driver/"
+    description = "DataStax C/C++ Driver for Apache Cassandra and DataStax Products"
+    topics = ("cassandra", "cpp-driver", "database", "conan-recipe")
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "install_header_in_subdir": [True, False],
+        "use_atomic": [None, "boost", "std"],
+        "with_openssl": [True, False],
+        "with_zlib": [True, False],
+        "with_kerberos": [True, False],
+        "use_timerfd": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "install_header_in_subdir": False,
+        "use_atomic": None,
+        "with_openssl": True,
+        "with_zlib": True,
+        "with_kerberos": False,
+        "use_timerfd": True
+    }
+
+    generators = "cmake"
+    exports_sources = [
+        "CMakeLists.txt",
+        "patches/*"
+    ]
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
```

Comment:
I suspect caching the cmake would be greatly beneficial in this case

Reply:
[Fixed](https://github.com/conan-io/conan-center-index/pull/3110/commits/4d96efddc722818534e3a515d0ff3ec73ec1e8d6)